lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -68,12 +68,22 @@
 """
 
 EXAMPLES = """
-- debug:
+- name: Return entire secret (a set of key-value pairs or a JSON object) from a path (which must not end in a slash)
+  debug:
+    msg: "{{ lookup('hashi_vault', 'secret=secret/hello token=c975b780-d1be-8016-866b-01d0f9b688a5 url=http://myvault:8200')}}"
+
+- name: Return one field with specific name within a secret.
+  debug:
     msg: "{{ lookup('hashi_vault', 'secret=secret/hello:value token=c975b780-d1be-8016-866b-01d0f9b688a5 url=http://myvault:8200')}}"
 
-- name: Return all secrets from a path
+- name: Return a list of all secrets under a given path (which must end in a slash)
   debug:
-    msg: "{{ lookup('hashi_vault', 'secret=secret/hello token=c975b780-d1be-8016-866b-01d0f9b688a5 url=http://myvault:8200')}}"
+    msg: "{{ lookup('hashi_vault', 'secret=secret/:keys token=c975b780-d1be-8016-866b-01d0f9b688a5 url=http://myvault:8200') }}"
```

Comment:
I'm not sure I understand how a list operation is semantically compatible with a `:keys` at the end.
I'd figure the list operation would not accept a field to list for each secret under a path.